src/firmware/app/primitives/test_util_world.h

```diff
@@ -0,0 +1,211 @@
+// This file is a base fixture for testing firmware primitives
+// The `fff` framework creates 'fake' functions for robot and world
+// "Fake" functions offer abilities like viewing args called and number of calls
+// To call a method, use [FUNCTION_NAME]_fake.[METHOD]
+// e.g. FAKE_VOID_FUNC(set_requested_rpm, int) -> set_requested_rpm_fake.arg0_val
+// `set_requested_rpm` takes in an integer. To see the argument called, call `arg0_val` on
+// the fake function
+// For more information, please visit the documentation:
+// https://github.com/meekrosoft/fff
```

Comment:
Could you clarify what you mean by "call a method"? 

`FAKE_VOID_FUNC(set_requested_rpm, int)` creates a fake function, and then we need to hit `set_requested_rpm_fake.arg0_val()` to get the value arg0 was when calling the fake function? 

Or is it just a struct that its accessing? I think the macro creates a struct, and u just access it values, with `set_requested_rpm_fake.arg0_val`

nit: could you re-format this to use the multiline comment `/** */`

Reply:
Reworded and fixed.